diff --git a/configure.ac b/configure.ac
index cd691dc..0658a7d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -51,7 +51,7 @@ AC_CHECK_HEADER([utf8cpp/utf8.h], [CPPFLAGS="-I/usr/include/utf8cpp/ $CPPFLAGS"]
 ])
 
 # Checks for POSIX thread support
-AX_PTHREAD([], [AC_MSG_ERROR([Can't find libpthread])])
+#AX_PTHREAD([], [AC_MSG_ERROR([Can't find libpthread])])
 
 # Checks for typedefs, structures, and compiler characteristics.
 AC_HEADER_STDBOOL
