diff --git a/lttoolbox/string_utils.cc b/lttoolbox/string_utils.cc
index 26356743..3bdf40a3 100644
--- a/lttoolbox/string_utils.cc
+++ b/lttoolbox/string_utils.cc
@@ -143,10 +143,9 @@ StringUtils::ftoa(double f)
 int
 StringUtils::stoi(const UString& str)
 {
-  int ret, len;
-  int c = u_sscanf(str.c_str(), "%d%n", &ret, &len);
-  // apparently %n isn't counted when calculating the return value of u_sscanf
-  if (c != 1 || static_cast<size_t>(len) != str.size()) {
+  int ret;
+  int c = u_sscanf(str.c_str(), "%d", &ret);
+  if (c != 1) {
     throw std::invalid_argument("unable to parse int");
   }
   return ret;
